com.example.WebPConverter.json

```diff
@@ -0,0 +1,62 @@
+{
+  "app-id": "com.example.WebPConverter",
```

Comment:
surely the upstream developers don't own `com.example`. The checkboxes for the question are not a CAPTCHA

Reply:
updated with my domain